eve/main.yml

```diff
@@ -368,7 +368,7 @@ stages:
         redis: enabled
         pykmip: enabled
         env:
-          <<: *mongo-vars
+          <<: *file-mem-mpu
```

Comment:
@rahulreddy

Reply:
It got ported over from the 7.x PRs but essentially KMIP tests are ok with either backends. They should be running both but they are very flaky with MongoDB, not sure if it's because of setup issues like pod being moved or something else.